### PR TITLE
Hooks: Add hooks for pyexcel and its plugins

### PR DESCRIPTION
**Pull request by HoLuLuLu**
Hooks: Add hooks for pyexcel and its plugins

Add hooks for pyexcel and the following plugins:
 - pyexcel-io
 - pyexcel-ods
 - pyexcel-ods3
 - pyexcel-odsr
 - pyexcel-xls
 - pyexcel-xlsx
 - pyexcel-xlsxw

### CODE REVIEW

File: news/4305.hooks.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Add hook for pyexcel
+Add hook for pyexcel-io
+Add hook for pyexcel-ods
+Add hook for pyexcel-ods3
+Add hook for pyexcel-odsr
+Add hook for pyexcel-xls
+Add hook for pyexcel-xlsx
+Add hook for pyexcel-xlsxw
```
Comment (htgoebel) [inline news/4305.hooks.rst]: Thanks for being so precise. OTOH for the readers of the release-notes "Added hook for pyexcel" would be enough information, What dou you think?

Comment (htgoebel) [reply news/4305.hooks.rst]: Or "Added hook for pyexcel and its plugins odsr, ...)

Comment (HoLuLuLu) [reply news/4305.hooks.rst]: You're right. I will change it.

**Comment by htgoebel** [comment]
Thanks for updating quickly. Unfortunately there are still some lint errors. I suggest running `flake8` on the new files.

**Comment by HoLuLuLu** [comment]
Now everything should be ok

**Comment by htgoebel** [comment]
Thanks for the pull-request and again for updating quickly.

**Pull request merged**